crates/parcel_plugin_transformer_js/src/transformer/conversion/symbol.rs

```diff
@@ -0,0 +1,170 @@
+use std::path::Path;
+
+use parcel_core::types::Symbol;
+
+use crate::transformer::conversion::loc::convert_loc;
+
+/// Convert `CollectImportedSymbol`, `ImportedSymbol` and into `Symbol`
+macro_rules! convert_symbol {
```

Comment:
Curious as to why this is a macro and not a function?

Reply:
Because the input types are different so for it to be a function I'd have to introduce a trait, implement, generic etc etc etc

Reply:
I don't love it;

Maybe it's better to repeat the code

Reply:
No that's fine and makes sense. Was just asking to learn.